process/smartContract/process.go

```diff
@@ -1150,6 +1063,17 @@ func (sc *scProcessor) processSimpleSCR(
 	return nil
 }
 
+// [NotConcurrentSafe]
```

Comment:
This doesn't seem to exist in Go. Better replace it with a `// TODO add mutex for last silent error`, or just ignore the concurrency concerns (transactions are executed serially). Parallel tests might fail though.

Reply:
Removed functions.